charts/pega/README.md

```diff
@@ -1045,15 +1045,15 @@ Parameter   | Description   | Default value
 ---         | ---           | ---
 `hazelcast.image` | Reference the `platform/clustering-service` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_HAZELCAST_IMAGE:TAG`
 `hazelcast.clusteringServiceImage` | Reference the `platform/clustering-service` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_CLUSTERING_SERVICE_IMAGE:TAG`
-`hazelcast.enabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.6, if not set the installation will fail | `true`
-`hazelcast.clusteringServiceEnabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.8, if not set the installation will fail | `false`
-`hazelcast.migration.enabled` |  Set as true after creating parallel cluster to migrate the data and client switch over to new Hazelcast service | `false`
-`hazelcast.migration.migrationJobImage` | Reference the `platform/clustering-service-kubectl` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
-`hazelcast.migration.embeddedToCSMigration` |  Set as true while creating parallel cluster in case of embedded source system | `false`
-`hazelcast.migration.skipRestart` |  Set as true while removing old hazelcast cluster after migration to avoid restart | `false`
-`hazelcast.replicas` | Number of initial members to join the Hazelcast cluster | `3`
-`hazelcast.username` | UserName to be used in client-server Hazelcast model for authentication | `""`
-`hazelcast.password` | Password to be used in client-server Hazelcast model for authentication | `""`
+`hazelcast.enabled` |  Set to `true` if client-server deployment of Pega Platform is required, otherwise leave set to `false`. Note: To avoid an installation failure, you must set this value to `false` for Pega platform deployments using versions before 8.6. | `true`
+`hazelcast.clusteringServiceEnabled` |  Set to `true` if client-server deployment of Pega Platform is required, otherwise leave set to `false`. Note: Set this value to `false` for Pega platform versions below 8.8, if not set the installation will fail. | `false`
+`hazelcast.migration.enabled` |  Set to `true` after creating parallel cluster (new Hazelcast) to establish the connection with platform and migrate the data. | `false`
+`hazelcast.migration.migrationJobImage` | Reference the `platform/clustering-service-kubectl` Docker image to create the migration job to run the migration script. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
+`hazelcast.migration.embeddedToCSMigration` |  Set to `true` while creating parallel cluster in case of embedded source system,  leave this value set to `false` in case of c/s to c/s migration. | `false`
```

Comment:
sorry but not clear yet...remove extra space before leave.
is might be a c/s to c/s migration, but what is the client actually doing in this case? why would they do such a migration...are they updating the HZCST version that is already externalized? it'd be best if we state directly the use case. I don't think c/s to c/s migration is clear...how ab out 
in the use case when clients with deployments already using the client-server models want to update the Hazelcast service version.

Reply:
@taz-pega-work , This is Hazelcast version migration , due to backward incompatibility we are creating a parallel service along with existing one and then doing the migration of data and connection switchover. Updated the description as below:
Set to `true` while migrating the data from existing embedded Hazelcast deployment to the new c/s Hazelcast deployment.